docs/notebooks/Training_visualizations.ipynb

```diff
@@ -0,0 +1,318 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Training Logs\n",
+    "\n",
+    "We can also log the metric values after every epoch to the shell apart from visualizing them in Visdom. The only change is to define `logger=\"shell\"` instead of `\"visdom\"` in the input callbacks."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:gensim.models.ldamodel:using symmetric alpha at 0.2\n",
+      "INFO:gensim.models.ldamodel:using symmetric eta at 0.0009950248756218905\n",
+      "INFO:gensim.models.ldamodel:using serial LDA version on this node\n",
+      "INFO:gensim.models.ldamodel:running online (multi-pass) LDA training, 5 topics, 3 passes over the supplied corpus of 25 documents, updating model once every 25 documents, evaluating perplexity every 25 documents, iterating 50x with a convergence threshold of 0.001000\n",
+      "WARNING:gensim.models.ldamodel:too few updates, training might not converge; consider increasing the number of passes or iterations to improve accuracy\n",
+      "INFO:gensim.models.ldamodel:-9.032 per-word bound, 523.4 perplexity estimate based on a held-out corpus of 25 documents with 2182 words\n",
+      "INFO:gensim.models.ldamodel:PROGRESS: pass 0, at document #25/25\n",
+      "INFO:gensim.models.ldamodel:topic #0 (0.200): 0.025*\"the\" + 0.022*\"to\" + 0.013*\"in\" + 0.011*\"of\" + 0.010*\"and\" + 0.007*\"s\" + 0.007*\"a\" + 0.005*\"said\" + 0.004*\"he\" + 0.004*\"iraqi\"\n",
+      "INFO:gensim.models.ldamodel:topic #1 (0.200): 0.054*\"the\" + 0.028*\"to\" + 0.023*\"of\" + 0.019*\"and\" + 0.017*\"in\" + 0.016*\"a\" + 0.013*\"s\" + 0.009*\"that\" + 0.008*\"has\" + 0.007*\"on\"\n",
+      "INFO:gensim.models.ldamodel:topic #2 (0.200): 0.049*\"the\" + 0.027*\"in\" + 0.024*\"to\" + 0.016*\"of\" + 0.012*\"and\" + 0.007*\"a\" + 0.007*\"that\" + 0.006*\"is\" + 0.006*\"for\" + 0.006*\"work\"\n",
+      "INFO:gensim.models.ldamodel:topic #3 (0.200): 0.041*\"the\" + 0.024*\"of\" + 0.021*\"to\" + 0.016*\"a\" + 0.016*\"and\" + 0.012*\"s\" + 0.010*\"in\" + 0.007*\"as\" + 0.006*\"with\" + 0.006*\"party\"\n",
+      "INFO:gensim.models.ldamodel:topic #4 (0.200): 0.034*\"the\" + 0.023*\"of\" + 0.016*\"a\" + 0.015*\"and\" + 0.012*\"in\" + 0.011*\"to\" + 0.008*\"as\" + 0.007*\"is\" + 0.007*\"with\" + 0.007*\"that\"\n",
+      "INFO:gensim.models.ldamodel:topic diff=1.878616, rho=1.000000\n",
+      "INFO:LdaModel:Epoch 0: Perplexity estimate: 495.948922721\n",
+      "INFO:LdaModel:Epoch 0: Perplexity estimate: 609.481073631\n",
+      "INFO:LdaModel:Epoch 0: Coherence estimate: -22.4407925538\n",
+      "INFO:LdaModel:Epoch 0: Diff estimate: [ 0.44194712  0.96670853  0.8036907   0.72372737  0.63141336]\n",
+      "INFO:LdaModel:Epoch 0: Convergence estimate: 0.0\n",
```

Comment:
Previous state of model (needed to calculate `Convergence`) was being saved at wrong place. Corrected it